discovery-provider/src/model_validator.py

```diff
@@ -0,0 +1,72 @@
+import json
+# import logging  # pylint: disable=C0302
+from jsonschema import validate, Draft7Validator, ValidationError, validators
```

Comment:
I know there's multiple drafts of json schema, haven't look too deep into it but wanted to ask why we are choosing draft07? I think it's the most supported?

Reply:
great question! so i thought draft07 is a good fit for us as it has functionality that supports our use case. it is the second most recent release (most recent is `2019-09`), and on top of the docs provided, i was able to find more context on different use cases online.